docs/GenericChart.md

```diff
@@ -0,0 +1,314 @@
+            {
+              "id": "time_step",
+              "source": "data"
+            }
+          ]
+        },
+        {
+          "id": "anc",
+          "label": "ANC",
+          "url": "/chart-data/input-time-series/anc",
+          "filters": [
+            {
+              "id": "plot_type",
+              "source": "data"
+            },
+            {
+              "id": "area_level",
+              "source": "data"
+            },
+            {
+              "id": "age",
+              "source": "data"
+            }
+          ]
+        }
+      ],
+      "dataSelectors": {
+	// Data source is currently the only type of data selector supported.
+	// A data source has a particular role in the chart. The user selects which dataset to use for each data source. 
+        "dataSources": [
+          {
+            // Id for this data source
+            "id": "data",
+
+	    // A data source might not be editable if only one dataset was available
+            "type": "editable", 
+
+            // Translation key for the label to use for the data source selector		
+            "label": "dataSource",  
+
+	    // Default dataset id to use for the data source		
+            "datasetId": "art",
+
+	    // Will be used to show or hide controls for user to select indicator (not yet implemented)
+            "showIndicators": false,
+
+	    // Show or hide controls for user to select filter values
+            "showFilters": true
+          }
+        ]
+      },
+      // Subplot configuration for the chart, if any. Used by GenericChart component to calculate the scroll height area.
+      "subplots": {
+	// Number of subplots to display per row
+        "columns": 3,
+
+	// The column in the data for which a subplot will be shown for each distinct value. 
+        "distinctColumn": "area_name",
+
+	// Height in pixels which should be allowed for each row of subplts.
+        "heightPerRow": 140
+      },
+      // This is an array as a future implementation may allow multiple chart types per chart e.g. scatter and bar. 
+      // Only one chart config per chart is currently supported		
+      "chartConfig": [
+        {
+          "id": "scatter",
+
+	  // When implemented, this is the translation key for the label to be displayed for this chart type's option 	
+          "label": "Scatter",
+		
+          "config": "...The JSONata config string goes here - see below for more details on this..."
+        }
+      ]
+    }
+
+```
+
+### Chart Config Jsonata
+
+Chart configuration is defined using [JSONata](https://jsonata.org/), a generic json transformation language - at runtime, the `Plotly`
+component invokes the transformation on the filtered chart data given the chart configuration JSONata to provide the 
+full Plotly chart configuration.
+
+The full Jsonata for the Input Time Series can be found 
+[here](https://github.com/mrc-ide/hint/blob/mrc-2537/src/app/src/main/resources/metadata/input-time-series-config-jsonata.txt),
```

Comment:
To be updated to master branch when this is merged

Reply:
```suggestion
[here](https://github.com/mrc-ide/hint/blob/mrc-2537/src/app/src/main/resources/metadata/input-time-series-config-jsonata.txt).
```